plugins/tmux/tmux.plugin.zsh

```diff
@@ -68,8 +71,9 @@ function _zsh_tmux_plugin_run() {
   # If failed, just run tmux, fixing the TERM variable if requested.
   if [[ $? -ne 0 ]]; then
     [[ "$ZSH_TMUX_FIXTERM" == "true" ]] && tmux_cmd+=(-f "$_ZSH_TMUX_FIXED_CONFIG") || \
-    [[ -e "$ZSH_TMUX_CONFIG" ]] && tmux_cmd+=(-f "$ZSH_TMUX_CONFIG")
-    $tmux_cmd new-session  
+    [[ -e "$ZSH_TMUX_CONFIG" ]] && tmux_cmd+=(-f "$ZSH_TMUX_CONFIG") || \
+    [[ -e "$ZSH_TMUX_CONFIG" ]] && tmux_cmd+=(" -u")
```

Comment:
Shouldn't this option be applied further up, like before the first tmux attach? I don't see it making sense here, after what is essentialy and else statement. Also, it checks `$ZSH_TMUX_CONFIG` instead of `$ZSH_TMUX_UNICODE`.

Reply:
You're totally right, my bad. That will teach me using a different zsh plugin manager :)